scripts/generate_dump_for_offline_apps.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+from __future__ import absolute_import, division, print_function
+import csv
+from itertools import imap
+from operator import itemgetter
+
+def main():
+    import os
+    if not (os.getenv('OFF_PUBLIC_DATA_DIR') and os.getenv('PRODUCT_OPENER_FLAVOR') and os.getenv('PRODUCT_OPENER_FLAVOR_SHORT')):
+        print("Environment variables OFF_PUBLIC_DATA_DIR, PRODUCT_OPENER_FLAVOR and PRODUCT_OPENER_FLAVOR_SHORT are required")
+        exit(1)
+    off_public_data_dir = os.getenv('OFF_PUBLIC_DATA_DIR')
+    product_opener_flavor = os.getenv('PRODUCT_OPENER_FLAVOR')
+    product_opener_flavor_short = os.getenv('PRODUCT_OPENER_FLAVOR_SHORT')
+
+    if not os.path.exists(off_public_data_dir + '/offline'):
+        os.makedirs(off_public_data_dir + '/offline')
+        
+    import pandas
+    df = pandas.read_csv(off_public_data_dir + '/' + product_opener_flavor + '/en.' + product_opener_flavor_short + '.products.csv', sep='\t', low_memory=False)
+    colnames = ['code','product_name','quantity','brands','nutriscore_grade','nova_group','ecoscore_grade']
```

Comment:
this won't work for non off flavors

Reply:
cf my other scripts, where I remove the final 3 columns

Reply:
the app can just ignore the columns it doesn't need